### DISCUSSION TITLE
Has there been a change in thumbnail preview sizes in admin?

### DISCUSSION THREAD

**Discussion by ainsleyclark** [payloadcms/payload#8874] [General]
Hey,

I'm not quite sure if it's me, or if there's been a change making thumbnail sizes in the admin considerably small from beta (around) `50` to the latest version.

**Latest**:
<img width="739" alt="image" src="https://github.com/user-attachments/assets/e320348e-2270-442d-abed-281dc52552a1">

**Previous**:
<img width="678" alt="image" src="https://github.com/user-attachments/assets/ca3db09f-c840-4a81-9d42-aae25c6fc9f4">

Is there a way to change this?

**Comment by Swahjak** [comment]
Looking for the same thing, it seems like the thumbnail size is currently hardcoded to https://github.com/payloadcms/payload/blob/f5aad49ba7d2fc159b71f91e8ebf37c012643c09/packages/ui/src/fields/Upload/RelationshipContent/index.tsx#L90 would be great if this we're configurable.

**Comment by gustavocardoso** [comment]
Any news? Any workarounds?

**Comment by ainsleyclark** [comment]
I don't know if the Payload team will consider this to implement as a feature (differing sizes for previews), but pretty sure you could customise the scss in the mean time.

There incredibly small, in terms of a gallery you have to click on it to find out what it is, so it kind of defeats the purpose of the preview.

**Reply by gustavocardoso** [reply]
Yeah. I agree with you.

I'm using css to increase the size of the thumbnail!

**Reply by MartinCura** [reply]
so does this still have to be configured by scss? it's not the biggest hassle, but it's weird overriding the "small" class to not be (as) small, and it makes me think that there should be a way to choose "medium" somewhere? anyway
